language-mapping.go

```diff
@@ -0,0 +1,50 @@
+package ctags
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+var SupportedLanguages = [...]string{"Basic", "C", "C#", "C++", "Clojure", "Cobol", "CSS", "CUDA", "D", "Elixir", "elm", "Erlang", "Go", "GraphQL", "Groovy", "haskell", "Java", "JavaScript", "Jsonnet", "kotlin", "Lisp", "Lua", "MatLab", "ObjectiveC", "OCaml", "Pascal", "Perl", "Perl6", "PHP", "Powershell", "Protobuf", "Python", "R", "Ruby", "Rust", "scala", "Scheme", "Sh", "swift", "SystemVerilog", "Tcl" /* "Thrift", */, "typescript", "tsx", "Verilog", "VHDL", "Vim"}
```

Comment:
why is thrift not in the list? Are you using the latest ctags when you tested? We recently removed our custom regexes for it since upstream ctags supports it now.

Reply:
In the old code we have thrift. In this code you have thrift commented out.

Reply:
Oh snap youre right yea, I had removed it because of ctags complaining but forgot 😬